src/main/java/uk/gov/hmcts/sptribs/ciccase/model/CaseData.java

```diff
@@ -61,6 +63,26 @@
 @Builder(toBuilder = true)
 public class CaseData {
 
+    @CCD(access = {DefaultAccess.class, CaseworkerWithCAAAccess.class},
+        typeOverride = Collection,
+        label = "Linked Cases",
+        typeParameterOverride = "CaseLink")
+    @Builder.Default
+    private List<ListValue<CaseLink>> caseLinks = new ArrayList<>();
+
+    @CCD(
+        label = "Component Launcher (for displaying Linked Cases data)",
+        access = {DefaultAccess.class, CaseworkerWithCAAAccess.class}
+    )
+    @JsonProperty("LinkedCasesComponentLauncher")
+    private ComponentLauncher linkedCasesComponentLauncher;
```

Comment:
This will need a migration as you have moved it from cicCase where all fields are prefixed with cicCase. Any field name changes require a migration if the field is already live in prod

Reply:
yes Tom. Migration of old cases will be done as part of another ticket. This PR is just for new cases.